src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -1133,41 +1135,8 @@ module Types = struct
                 coinbase_receiver ) ] )
 
   let protocol_state_proof : (Coda_lib.t, Proof.t option) typ =
-    let display_g1_elem (g1 : Crypto_params.Tick_backend.Inner_curve.t) =
-      let x, y = Crypto_params.Tick_backend.Inner_curve.to_affine_exn g1 in
-      List.map [x; y] ~f:Crypto_params.Tick0.Field.to_string
-    in
-    let display_g2_elem (g2 : Curve_choice.Tock_full.G2.t) =
-      let open Curve_choice.Tock_full in
-      let x, y = G2.to_affine_exn g2 in
-      let to_string (fqe : Fqe.t) =
-        let vector = Fqe.to_vector fqe in
-        List.init (Fq.Vector.length vector) ~f:(fun i ->
-            let fq = Fq.Vector.get vector i in
-            Crypto_params.Tick0.Field.to_string fq )
-      in
-      List.map [x; y] ~f:to_string
-    in
-    let string_list_field ~resolve =
-      field
-        ~typ:(non_null @@ list (non_null string))
-        ~args:Arg.[]
-        ~resolve:(fun _ (proof : Proof.t) -> display_g1_elem (resolve proof))
-    in
-    let string_list_list_field ~resolve =
-      field
-        ~typ:(non_null @@ list (non_null @@ list @@ non_null string))
-        ~args:Arg.[]
-        ~resolve:(fun _ (proof : Proof.t) -> display_g2_elem (resolve proof))
-    in
-    obj "protocolStateProof" ~fields:(fun _ ->
-        [ string_list_field "a" ~resolve:(fun (proof : Proof.t) -> proof.a)
-        ; string_list_list_field "b" ~resolve:(fun (proof : Proof.t) -> proof.b)
-        ; string_list_field "c" ~resolve:(fun (proof : Proof.t) -> proof.c)
-        ; string_list_list_field "delta_prime"
-            ~resolve:(fun (proof : Proof.t) -> proof.delta_prime)
-        ; string_list_field "z" ~resolve:(fun (proof : Proof.t) -> proof.z) ]
-    )
+    (* TODO *)
```

Comment:
Yes, this can be opaque -- the bit I said about backwards compatibility I think is not important for this part since I'm certain no one is depending on it yet (other than our old snark demo, which is broken now for other reasons anyway). In fact, we can just kill this obj entirely probably.